### DISCUSSION TITLE
Airflow DAG render option

### DISCUSSION THREAD

**Discussion by Lee2532** [apache/airflow#36033] [Q&A] [answered]
I don't parse DAG files on airflow scheduler, so is there a way to make them invisible on web server?

When using the current DAG temporarily or in the medium term, I think it is inefficient to annotate the entire DAG file or delete the file. Is there a particular parameter that is not exposed on the web server?

**Comment by potiuk** [comment]
Web server does not need DAG files at all. In Airflow Chart webserver POD does not have the DAG folder mounted.

The 'DAG source" in Airflow UI is read from the DB - when DAG File Processor parses it, it will save the content of that DAG file to the DB.

You can remove the old DAG entries in the db clean command I believe if that's what you are after. 

Though I am not sure what your "expose" and "inefficient" means, also "annotate the entire DAG file or delete it" is pretty cryptic. If you explain what you are after, maybe the answer will be more helpful.

**Reply by Lee2532** [reply]
There are many DAG files that are disabled on the web server, so I want to clean them up, but some DAGs use them intermittently. Therefore, I want to modify specific parameters and expose them to the web server only when necessary. (Not Deactivated)

To do this, the method I'm currently doing is either annotating the entire DAG file or deleting it.
Is there any other way?

**Reply by potiuk** [reply, marked as answer]
You can use .airflowignore.  Look for it in the documentation

**Reply by Lee2532** [reply]
Right, why didn't I think about that
